demo/Makefile

```diff
@@ -59,9 +63,11 @@ create_topic_no_ts_by_size:
 		kafka-topics \
 		--bootstrap-server $(bootstrap_server) \
 		--create \
+		--config retention.ms=-1 \
 		--config segment.bytes=$(segment) \
```

Comment:
What does `no_ts_by_size` mean? Is it by size or by time? From the code I can guess that it's by size, but the name is weird in that case. Same in other similarly named tasks

Reply:
sure, applying suggestions. have a look.